Samples/MobileBuyIntegration/MobileBuyIntegration/AppConfiguration.swift

```diff
@@ -38,6 +38,22 @@ public final class AppConfiguration: ObservableObject {
 
     // Displays the Checkout with ApplePay button
     @Published var applePayEnabled: Bool = true
+
+    /// AcceleratedCheckouts configuration
+    var acceleratedCheckoutsConfiguration: ShopifyAcceleratedCheckouts.Configuration {
+        ShopifyAcceleratedCheckouts.Configuration(
+            storefrontDomain: storefrontDomain,
+            storefrontAccessToken: InfoDictionary.shared.accessToken
+        )
+    }
+
+    /// ApplePay configuration for AcceleratedCheckouts
+    var applePayConfiguration: ShopifyAcceleratedCheckouts.ApplePayConfiguration {
+        ShopifyAcceleratedCheckouts.ApplePayConfiguration(
+            merchantIdentifier: "merchant.com.shopify.example.MobileBuyIntegration.ApplePay",
+            contactFields: [.email, .phone]
+        )
+    }
```

Comment:
Configuration for Apple Pay and Shop pay in the top-level App Configuration for the sample